src/components/homePage.tsx

```diff
@@ -0,0 +1,102 @@
+import HomeCard from "./common/homeCard";
+import CustomImage from "./common/customimage";
+import OPM from "../views/opm";
+import OpmComparison from "../views/opmComparison";
+import avg_orders_per_min from "../assets/avg_orders_per_min.svg";
+import total_no_of_orders from "../assets/total_no_of_orders.svg";
+import total_order_comp from "../assets/total_order_comp.svg";
+import avg_opm_comp from "../assets/avg_opm_comp.svg";
+import trending_down from "../assets/trending_down.svg";
+
+const CardTitle = ({ title, icon }) => {
+  return (
+    <div className="flex justify-between">
+      <h6>{title}</h6>
+      <CustomImage src={icon} />
+    </div>
+  );
+};
+
+const CardValue1 = ({ value, time }) => {
```

Comment:
Try a different name instead of CardValue1 and CardValue2

Reply:
changed